test/e2e/tests/tokens/custom-token-send-transfer.spec.js

```diff
@@ -346,13 +340,10 @@ describe('Transfer custom tokens @no-mmi', function () {
 
           // check token amount is correct after transaction
           await clickNestedButton(driver, 'Tokens');
-          const tokenAmount = await driver.findElement(
-            {
-              css: '[data-testid="multichain-token-list-item-value"]',
-              text: '8.5 TST',
-            },
-            { timeout: 10000 },
-          );
+          const tokenAmount = await driver.findElement({
+            css: '[data-testid="multichain-token-list-item-value"]',
+            text: '8.5 TST',
+          });
           assert.ok(tokenAmount, 'Token amount is not correct');
```

Comment:
maybe we can remove this legacy code, as it's unnecessary, the previous step will make sure the element is there or fail, what do you think?

Reply:
totally ! addressed here https://github.com/MetaMask/metamask-extension/pull/29235/commits/cf0692c129e2fe03b1f1f880c4a369de44f5c99a